infrastructure/lambda-nems-message.tf

```diff
@@ -0,0 +1,71 @@
+module "nems-message-lambda" {
+  count          = local.is_mesh_forwarder_enable ? 1 : 0
+  source         = "./modules/lambda"
+  name           = "NemsMessageLamba"
+  handler        = "handlers.nems_message_handler.lambda_handler"
+  lambda_timeout = 60
+  iam_role_policies = [
+    "arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole",
+    "arn:aws:iam::aws:policy/CloudWatchLambdaInsightsExecutionRolePolicy",
+    module.lloyd_george_reference_dynamodb_table.dynamodb_policy,
+  ]
```

Comment:
Maybe we should attach an IAM policy to allow this lambda to read messages from `module.sqs-nems-queue`?